tooltips: Use tippy tooltips for drafts and schedule message overlays.

Ddharmani3 opened the pull request:
Fixes #25892

<!-- Describe your pull request here.-->

Fixes: <!-- Issue link, or clear description.-->

<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**
![image](https://github.com/zulip/zulip/assets/64723994/f48aa8a7-5081-4286-9b63-7f0b4b4ddc1c)

![image](https://github.com/zulip/zulip/assets/64723994/03d2a1b8-fffa-4f13-a5c9-7562b75e4820)


<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans (e.g., issue description).
- [x] Highlights technical choices and bugs encountered.
- [x] Calls out remaining decisions and concerns.
- [x] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

Ddharmani3 commented:
@zulipbot add "buddy review"

timabbott commented:
I think the tooltips are fine, but we need `cursor: pointer` to be set -- it feels weird for it to not be obviously a click target.

Ddharmani3 commented:
> I think the tooltips are fine, but we need `cursor: pointer` to be set -- it feels weird for it to not be obviously a click target.

@timabbott, addressed the feedback, this PR is ready for review now.

timabbott commented:
Merged, thanks @Ddharmani3!
